storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -227,6 +231,59 @@ public static boolean showSkipOnCompactNotification() {
         return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
     }
 
+    public static boolean showNextChapterOnCompactNotification() {
+        return showButtonOnCompactNotification(NOTIFICATION_BUTTON_SKIP);
+    }
+
+    public static List<Integer> getFullNotificationButtons() {
+        String[] buttons = TextUtils.split(
+            prefs.getString(PREF_FULL_NOTIFICATION_BUTTONS,
+                NOTIFICATION_BUTTON_REWIND + ","
+                    + NOTIFICATION_BUTTON_FAST_FORWARD + ","
+                    + NOTIFICATION_BUTTON_SKIP + ","
+                    + NOTIFICATION_BUTTON_NEXT_CHAPTER + ","
+                    + NOTIFICATION_BUTTON_PLAYBACK_SPEED),
```

Comment:
The default should not be to show all buttons. I would actually make the default to have only skip, rewind, fast forward, and play.